.github/workflows/main.yml

```diff
@@ -73,7 +81,10 @@ jobs:
       env:
         GHPAGES_SSH_KEY: ${{ secrets.GHPAGES_SSH_KEY }}
       if: github.event_name != 'pull_request'
-    - uses: shawnbot/action-status@master
+    - if: >-
+        github.event_name != 'pull_request' &&
+        github.repository == 'planetarium/libplanet'
+      uses: shawnbot/action-status@master
```

Comment:
`L84:86` remain because I couldn't find to silent this step on fork repo.

Reply:
The error you'd encountered was this, right?:

    cat: Docs/obj/url.txt: No such file or directory

The reason why that file does not exist is that the previous step (`Docs/publish.sh`) hasn't executed (`if: github.event_name != 'pull_request'`).  Therefore, it would be find if these two steps have the same condition.

Reply:
The (`Docs/publish.sh`) step has executed but the (`hawnbot/action-status@master`) step failed due to empty token (`GH_CHECK_STATUS_TOKEN`). 

I replaced above condition (`github.event_name != 'pull_request' && github.repository == 'planetarium/libplanet') to check empty token (`github.event_name != 'pull_request' && env.GH_CHECK_STATUS_TOKEN != ''`)